src/com/pentalog/pentastagiu/homework3/ex2/Basket.java

```diff
@@ -0,0 +1,51 @@
+package com.pentalog.pentastagiu.homework3.ex2;
+
+public class Basket {
+    private double totalPrice;
+    private Costumer costumer;
+
+    public Basket(Costumer costumer) {
+        this.costumer = costumer;
+    }
+
+    private void addProduct(Product produs) {
+        this.totalPrice += produs.getPrice();
+        System.out.println("The product was successfully added!! Total price = " + this.totalPrice + " lei.");
+    }
+
+    private void infoBasket() {
```

Comment:
la fel si aici, sugerez metode publice pentru a putea fi refolosita implementarea ta in toata aplicatia